src/utils/validationUtil.ts

```diff
@@ -569,4 +573,39 @@ export class ValidationUtil {
     //     throw new models.HTTPError(400, 'Failed IVA tests must have IVA Defects');
     //   }
   }
+
+  /**
+   * Validates central docs for an array of test types
+   * @param testTypes TestType[]
+   * @throws HTTPError with status 400 if validation fails
+   */
+  public static validateCentralDocs(testTypes: TestType[]): void {
+    testTypes.forEach((testType) => {
+      if (
+        TestTypeHelper.validateTestTypeIdInList(
+          CENTRAL_DOCS_TEST,
+          testType.testTypeId,
+        ) &&
+        !testType?.centralDocs
+      ) {
+        throw new models.HTTPError(
+          400,
+          `Central docs required for test type ${testType.testTypeId}`,
+        );
+      }
+      if (
+        !TestTypeHelper.validateTestTypeIdInList(
+          CENTRAL_DOCS_TEST,
+          testType.testTypeId,
+        ) &&
+        testType?.centralDocs
+      ) {
+        throw new models.HTTPError(
+          400,
+          `Central docs is not required for test type ${testType.centralDocs}`,
```

Comment:
Should this be 
`Central docs required for test type ${testType.testTypeId}` rather than `${testType.centralDocs}` ?

Reply:
is 'required' the right terminology here anyway?

Arent we really saying that documents cant be generated centrally for that type? rather than them just not being required?

Reply:
So `Central docs is not available for test type: ${testType.testTypeId}` ?

Reply:
We are saying that it can't be generated for that test type yes. Would 
          `Central documents can not be issued for test type ${testType.testTypeId}`,

Reply:
that reads better